doc/internal/man3/ossl_punycode_decode.pod

```diff
@@ -10,27 +10,26 @@ ossl_punycode_decode, ossl_a2ulabel, ossl_a2ucompare
   #include "crypto/punycode.h"
 
   int ossl_punycode_decode(const char *pEncoded, const size_t enc_len,
-	unsigned int *pDecoded, unsigned int *pout_length);
+  unsigned int *pDecoded, unsigned int *pout_length);
```

Comment:
These are arguments to `ossl_punycode_decode()`, but indented like this, they are too easy to mistake for a new function.  Please push it to the right to be aligned with the opening parenthesis.

Reply:
Many thanks! Done.